packages/webview_flutter/webview_flutter/lib/src/webview.dart

```diff
@@ -268,6 +269,11 @@ class WebView extends StatefulWidget {
   /// By default `userAgent` is null.
   final String? userAgent;
 
+  /// A Boolean value indicating whether the WebView should support zooming using its on-screen zoom controls and gestures.
```

Comment:
nit: Please wrap to 80 characters.

Reply:
Fixed in a1dbd36, although I think the `format` step on CI barked at me when I tried to wrap these lines before?